backend/src/test/java/de/neuefische/driver/backend/services/DeliveryServiceTest.java

```diff
@@ -14,12 +14,20 @@
 import static org.mockito.Mockito.verify;
```

Comment:
```suggestion
import static org.mockito.Mockito.*;
```
dann muss man nicht 3 mal mockito importieren

Reply:
thx